### ISSUE TITLE
Estimating Training Data Influence by Tracking Gradient Descent

### ISSUE THREAD

**Issue by silky** [BraneShop/showreel#472]
![image](https://user-images.githubusercontent.com/129525/75082837-af591780-5569-11ea-8f9f-c27cde29f175.png)

https://arxiv.org/pdf/2002.08484.pdf
